### DISCUSSION TITLE
Construction Paper Experiment

### DISCUSSION THREAD

**Discussion by sabrina-pillai** [STELLA-Landsat/STELLA#38] [For Educators]
Place different color sheets of construction paper on a window or in direct sunlight and see which colors fade the fastest based on the visible light results (blue should fade fastest, and red the slowest)

**Comment by Landsat-Mike** [comment]
What is the STELLA tie in? Would we use the STELLA on the construction paper to get readings as time progressed? Should we try to use gradients of colors as well?
